### PR TITLE
Implemented chunked uploads

### PR DESCRIPTION
**Pull request by artificiality**
Throws exception for POST requests greater than 10485760 bytes due to _max-post-size_ limit in WildFly

### CODE REVIEW

File: src/main/webapp/filemanager.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
    <script src="dist/jquery.min.js"></script>
    <script src="dist/angular.min.js"></script>
    <script src="dist/angular-translate.min.js"></script>
-   <script src="dist/ng-file-upload.min.js"></script>
+   <script src="dist/ng-file-upload.js"></script>
```
Comment (syndoc) [inline src/main/webapp/filemanager.html]: We should use minified version in checked in code. Is there a reason why this file is not minified?

**Comment by syndoc** [comment]
Regarding "Throws exception for POST requests greater than 10485760 bytes due to max-post-size limit in WildFly" - did you test with updated WildFly configuration by increasing max-post-size ? Its documented at https://issues.jboss.org/browse/UNDERTOW-185.

**Comment by syndoc** [comment]
I did not see any attached test case, I think, there should be a test case to upload file and then list files to see if file got uploaded.

File: src/main/webapp/filemanager.html (hunk 1 above)
Comment (artificiality) [reply src/main/webapp/filemanager.html]: Had changed it for debugging and made a few changes. Fixed.

**Pull request merged**